frontend/public/locales/en/public.json

```diff
@@ -419,6 +419,10 @@
   "ReplicationController details": "ReplicationController details",
   "Phase": "Phase",
   "{{statusReplicas}} of {{specReplicas}} pods": "{{statusReplicas}} of {{specReplicas}} pods",
+  "Label": "Label",
+  "app=frontend": "app=frontend",
```

Comment:
This shouldn't be externalized since it's an actual label selector. It can't be translated.